test/decl/call/protocol.swift

```diff
@@ -1,27 +1,26 @@
 // RUN: %target-typecheck-verify-swift
 
 protocol P0 {
-  // expected-note @+1 {{protocol requires 'call' method with type '() -> Missing'; do you want to add a stub?}}
-  call func() -> Self
+  // expected-note @+1 {{protocol requires function 'call()' with type '() -> Missing'; do you want to add a stub?}}
+  func call() -> Self
 }
 func testProtocol(_ x: P0) {
   _ = x()
 }
-func testGeneric<T : P0>(_ x: T) {
```

Comment:
Why is this test removed? Callable via protocol conformance constraints should be tested.

Reply:
Sorry, removing that test was an accident. Re-added.
Also, renamed/moved tests to `test/Sema` from `test/decl/call`.